Update interfaces/SVG.idl

autofoolip opened the pull request:
This PR was automatically created by a bot.

Before merging, please check that any tests that depend on the updated IDL files still work.

If additional changes are needed, please manually create another PR based on this one.

See the [README](https://github.com/web-platform-tests/wpt/blob/master/interfaces/README.md) for how the IDL files in this directory are used.

<hr>

Note: This file was recently manually updated in commit 5f4cbbb164.
      This commit may revert some of those changes.

Source: https://github.com/tidoust/reffy-reports/blob/39f92a5/whatwg/idl/SVG.idl
Build: https://travis-ci.org/tidoust/reffy-reports/builds/447306145

foolip commented:
Ah, so this PR was created because https://github.com/web-platform-tests/wpt/pull/13740 updated wpt but the spec wasn't updated. I've sent https://github.com/w3c/svgwg/pull/564.

autofoolip committed:
Update interfaces/SVG.idl

Note: This file was recently manually updated in commit 5f4cbbb164.
      This commit may revert some of those changes.

Source: https://github.com/tidoust/reffy-reports/blob/39f92a5/whatwg/idl/SVG.idl
Build: https://travis-ci.org/tidoust/reffy-reports/builds/447306145
